src/pages/settings/PasswordPage.js

```diff
@@ -1,25 +1,158 @@
-import React from 'react';
+import React, {Component} from 'react';
+import {
+    View,
+    TextInput,
+} from 'react-native';
+import Onyx, {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import {isEmpty} from 'underscore';
+
 import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
 import Navigation from '../../libs/Navigation/Navigation';
 import ROUTES from '../../ROUTES';
 import ScreenWrapper from '../../components/ScreenWrapper';
+import Text from '../../components/Text';
+import styles from '../../styles/styles';
+import ONYXKEYS from '../../ONYXKEYS';
+import ButtonWithLoader from '../../components/ButtonWithLoader';
+import {changePassword} from '../../libs/actions/User';
+
+const propTypes = {
+    /* Onyx Props */
+    // Holds information about the users account that is logging in
+    account: PropTypes.shape({
+        // Whether 2FA is required for the users account that is logging in
+        requiresTwoFactorAuth: PropTypes.bool,
```

Comment:
I wasn't able to test this prop. I turned on 2FA in expensify.com but it would always return false for me.

Also the change password request came successful, but the followup call to Auth would fail with `402 Requires Two Factor Auth`

Reply:
I was able to get `requiresTwoFactorAuth` = `true`
<img width="200" alt="" src="https://user-images.githubusercontent.com/16747822/111197792-1e8e1900-857c-11eb-9c9d-ef1730b11b19.png">

View looks good, and confirmed that I can change my password successfully with the correct 2FA code.
<img width="300" alt="" src="https://user-images.githubusercontent.com/16747822/111197877-382f6080-857c-11eb-8538-b0c7cc516e89.png">

However, I think we need a way to validate the 2FA code before actually changing the password (otherwise you just get booted out immediately if the 2FA code is wrong since we can't authenticate you). We have an API command `TwoFactorAuth_Validate` that takes in a `twoFactorAuthCode` that we haven't added to `API.js` yet. You could add that and call it `User.changePassword` if we pass along a 2fa code.

cc: @bondydaa since you're our resident 2FA expert

Reply:
Weird - it seems to works for me. Did you try signing out of .cash after enabling 2FA on expensify.com?

<img width=300 src="https://user-images.githubusercontent.com/2229301/111208105-957cdf00-8587-11eb-91bf-1e755217c839.png" />

Reply:
Also looking into an alternative method so that we could use `API.reauthenticate` and not have to require the 2FA code at all.

Reply:
OK found something - it'll require an Auth PR (which will take a day or two to get deployed probably) but will fix the issue such that we can do what I mentioned above and call `API.reauthenticate('ChangePassword');` instead of `signIn` and also not require the user to enter a 2FA code at all.

Reply:
Yup, looks like I needed to logout/login to get the updated 2FA value

Reply:
OK cool, after a long investigation it turns out we don't need to call `signIn` or `reauthenticate` when changing the password. Nor do we need an auth PR for this. Nor do we need to get the 2FA code for changing the password. 

@Maftalion could you remove the 2FA portion of the form from the Change Password view and also remove `signIn(password, twoFactorAuthCode);` from `User.changePassword`? Sorry for the confusion here.